policy-controller/src/main.rs

```diff
@@ -166,6 +173,13 @@ async fn main() -> Result<()> {
         runtime.shutdown_handle(),
     ));
 
+    // Spawn the status controller, which will process patches from the index
+    // and apply them to resources.
+    let client = runtime.client();
+    tokio::spawn(async move {
+        status_controller::process_patches(client, patches_rx).await;
+    });
```

Comment:
tiny nit: we could remove the `async { ... }.await` here; it's equivalent to just write:
```suggestion
    tokio::spawn(status_controller::process_patches(client, patches_rx));
```